### DISCUSSION TITLE
getAllUserInfo() 호출 후 모두멘션

### DISCUSSION THREAD

**Discussion by disabled-variable** [storycraft/node-kakao#688] [General]
getAllUserInfo()로 해당 채팅방의 멤버들의 정보를 받아온뒤
모두멘션을 하려면 어떻게 해야하나요?

**Comment by tadashi74** [comment]
```
 if (data.text === '/everyone') {
        const builder = new ChatBuilder();
        for (const user of channel.getAllUserInfo()) {
            builder.append(new MentionContent(user)).text(' ')
        }
        channel.sendChat(builder.build(KnownChatType.TEXT));
  }
```

기본 소스는 여기있습니다. 혹시 텍스트상으로는 멘션이 안된것처럼 보이는데 모두 멘션됬다고 알림가는 소스 가지신 분 계실까요?

**Reply by generalxd** [reply]
이거 아닌가요 ?

**Reply by dbswlstjr2** [reply]
네

**Comment by ghost** [comment]
카카오측에서 어뷰징 방지를 위해 카카오톡 클라이언트에 15명 까지 멘션이 가능하게 하고 이후 서버측 패치로 15명 이상의 멘션을 막은듯 합니다.
![IMG_20211002_032256](https://user-images.githubusercontent.com/24631476/135669005-a398aa58-b106-4e3b-aa8a-fa87f96fae3b.jpg)

**Reply by dbswlstjr2** [reply]
15명씩 끊어서 해보세요...
